rllib/core/rl_module/tests/test_rl_module_specs.py

```diff
@@ -97,14 +70,14 @@ def test_customized_multi_agent_spec(self):
 
         # TODO (Kourosh): add tf support
         for fw in ["torch"]:
-            spec_cls = CUSTOM_MARL_SPECS[fw]
-            module_cls = CUSTOM_MODULES[fw]
+            marl_module_cls = CUSTOM_MARL_MODULES[fw]
```

Comment:
we should just call this test_customized_multi_agent_module since there are no longer custom specs.

Reply:
done.